README.md

```diff
@@ -1,54 +1,367 @@
+should configure WOVN.php as below.
+```
+api_timeout = 1
+```
+
+#### `disable_api_request_for_default_lang`
+This parameter tells WOVN.php whether or not it should use our localization API
+when content is requested in original language. By default, the
+`disable_api_request_for_default_lang` option is set to `0` (fasle). It means
+that WOVN.php will use our localization API even if the content does not have to
+be translated. When this setting is set to `1`, you may notice more server
+resource being used. This is because WOVN.php has to do some HTML parsing that
+our localization API usually does (for instance, to insert `hreflang`
+information). However, it will save web page loading time since it does not send
+a request to our API. If you experience no resource issues, we recommand you to
+deactivate API requests for original language as below.
+```
+disable_api_request_for_default_lang = 1
 ```
-require: {
-    "WOVNio/WOVN.php": "*"
-}
+
+#### `use_proxy`
+This parameter tells WOVN.php whether or not your content is served through a
+proxy. By default, this setting is set to `0` (false). If your content is served
+through a proxy, WOVN.php needs to know it when gathering information based on
+requested URL. In that case, you should set `use_proxy` to `1` (true).
 ```
-### 2. Edit the configuration file
-These are the two main settings for your project
- - project_token: your WOVNio's project token
- - url_pattern: query, path or subdomain (default is query)
- - default_lang: the code of the default language of your website, ex: ja (for Japanese)
- - encoding: encoding of html. If not specified, detect automatically.
-     Supported encoding: 'UTF-8', 'EUC-JP', 'SJIS', 'eucJP-win', 'SJIS-win', 'JIS', 'ISO-2022-JP', 'ASCII'
+use_proxy = 1
+```
+
+#### `override_content_length`
+This parameter tell WOVN.php whether or not it should update the response header
+"Content-Length". By default, `override_content_length` is set to `0` (false).
+If you need to update the response header "Content-Length" you should set
+`override_content_length` to `1` (true).
+```
+override_content_length = 1
+```
+
+## 4. Bug Report
+If you encounter problems installing WOVN.php, setting WOVN.php, or you just
+found a bug on your website that is related to WOVN.php, please contact us at
+[support@wovn.io](support@wovn.io). To help us finding a solution to your
+issue we will require some information. First we need to know on which web pages
+you encounter the issue, as well as steps to reproduce. If possible, we would
+also need test accounts if your issues occur on web pages behind authorization
+(we recommend you to use a staging server for that matter).
 
-Please copy the sample file from the library `wovn.ini.sample`, rename it as `wovn.ini` and paste it on your framework root directory (same level as WOVN.php directory)
--->
+If your problems are happening on your server side (widget snippet not inserted,
```

Comment:
should we call this something else than "widget snippet"? ("wovn script tag", for example?)

Reply:
Change to "widget `<script>` tag"